drivers/input/i8042prt/pnp.c

```diff
@@ -263,6 +263,52 @@ i8042DetectMouse(
     INFO_(I8042PRT, "Mouse not detected\n");
 }
 
+static void
+i8042DetectMouseType(
+    IN PPORT_DEVICE_EXTENSION DeviceExtension)
+{
+    UCHAR Value;
+    INT i;
```

Comment:
Please add a newline after this variable to separate the variables from the rest of the code.